jetstream/pre_treatment.py

```diff
@@ -28,6 +28,11 @@ def apply(self, df: DataFrame, col: str) -> DataFrame:
         """
         raise NotImplementedError
 
+    @classmethod
+    def from_dict(cls, config_dict: Dict[str, Any]):
+        """Create a class instance with the specified config parameters."""
+        return cls(**config_dict)  # type: ignore
```

Comment:
can we use cattrs to handle this instead?

Reply:
I guess we could. I thought it might be good to be in line with `Statistics.from_dict` here

Reply:
Yeah, it's probably better to be stricter.

Reply:
(i.e. this way seems better than using cattrs)